core/unix/os.c

```diff
@@ -8465,6 +8465,21 @@ post_system_call(dcontext_t *dcontext)
     dcontext->whereami = old_whereami;
 }
 
+#ifdef STATIC_LIBRARY
+/* static libraries may optionally define two linker variables
+ * (dynamorio_so_start and dynamorio_so_end) to help mitigate
+ * edge cases in detecting DR's library bounds.
+ *
+ * If not specified, the variables' location will default to
+ * weak_dynamorio_so_bounds_filler and they will not be used.
+ */
+static int weak_dynamorio_so_bounds_filler;
+extern int dynamorio_so_start
+    __attribute__((weak, alias("weak_dynamorio_so_bounds_filler")));
```

Comment:
IIRC Mac has no alias support so this would need to be `ifdef LINUX.`  (Mac is run post-commit on Travis -- can enable pre temporarily by tweaking `.travis.yml`.)  The linker vars are Linux-only in any case so doubly so.

Reply:
Thanks for catching before submission - fixed all of this to be STATIC_LIBRARY and LINUX.